doc/lspconfig.txt

```diff
@@ -631,6 +631,18 @@ options, the `lspconfig` wiki lists community created plugins that build upon
 the built-in client to provide functionality tailored to specific language
 servers.
 
+==============================================================================
+Highlights                                                *lspconfig-highlight*
+
+LspInfoTitle        Client name
+LspInfoList         Server name list
+LspInfoFiletype     `filetypes` area
+LspInfoTip          Tip
+LspInfoBorder       Window border
```

Comment:
This is the wrong direction. nvim-lspconfig should strive to be simple. Adding custom highlight groups , which requires documentation and backwards compatibility, is out of scope.

Reply:
this is correct. lspconfig only provides basic server configuration, about the highlight of the window. Predefined highlights have been used in the past. It's not that prominent in some themes. And the user doesn't have any options for modification. The only way is to modify the theme. This sucks. So let's make them optional in this pr.  
Use a new highlight group from a predefined highlight group. then link them.  This pr doesn't add too much code. Do not worry.

Reply:
yup. Because usually this command is used when debugging. Maybe users don't care. But I just want it to work fine in any configuration. .maybe it's a reckless pr . Sry